gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
```

Comment:
@danimontoya this version is used by Arrow Meta. It's not involved in this case.

There are 2 versions that should have the same value:
* The version in that file in `arrow-typeproofs` repository
* `BASE_GRADLE_VERSION` in this repository which is used to run the test on `arrow-typeproofs` repository. That's the reason why they must be the same version.

Reply:
I thought so.. I did it because just to give another try. Tried already with BASE_GRADLE_VERSION and gradle version from typeproofs being the same (6.5.1) and didn't work
I will revert that next! Thanks!! 😄